package.json

```diff
@@ -6,6 +6,8 @@
   "dependencies": {
     "@oclif/command": "^1",
     "@oclif/config": "^1",
+    "ink": "^2.7.1",
+    "react": "^16.13.1",
```

Comment:
There was an update on the test command. Check the `next-release` branch on toolbelt